api_app/tests_ma/test_api/test_routes/test_airlock.py

```diff
@@ -109,6 +109,7 @@ def sample_review_resource(healthy=True) -> UserResource:
         templateVersion="0.0.1",
         _etag="123",
         isEnabled=True,
+        properties={"azure_resource_id": "abc"},
```

Comment:
I know this is just the test but when / why have we started to persist azure resource ids?

Reply:
Anyways, could we push this to a Teams conversation if you have any concerns on that and get this PR through so it unblocks people's UTs?